src/main/webapp/script.js

```diff
@@ -168,14 +169,19 @@ function addReasonInput() {
   reason.setAttribute("name", "reason");
   reason.setAttribute("placeholder", "Enter reason for use.");
   reason.setAttribute("required", "true");  
+  reason.focus();//makes a glowing border on focus/hover
```

Comment:
I really liked the glow when I was testing this out!

Reply:
Same!